lib/net_mosq.c

```diff
@@ -537,6 +544,10 @@ int net__socket_connect_tls(struct mosquitto *mosq)
 {
 	int ret, err;
 
+    if (!is_tls_initialized) {
+	    net__init_tls_engine();
+    }
+
```

Comment:
Could you please replace spaces here with a tab.

Reply:
oops... changed